source/common/http/BUILD

```diff
@@ -12,8 +12,14 @@ envoy_package()
 
 envoy_cc_library(
     name = "async_client_lib",
-    srcs = ["async_client_impl.cc"],
-    hdrs = ["async_client_impl.h"],
+    srcs = [
+        "async_client_impl.cc",
+        "null_route_impl.cc",
```

Comment:
Please create a separate target for the null_route_impl.cc library, since it is canonical approach with bazel.

Reply:
done, thanks!